Classes/item.rb

```diff
@@ -0,0 +1,23 @@
+require 'date'
+
+class Item
+  attr_accessor :archived, :label, :author, :id, :publish_date, :genre
+
```

Comment:
- [ ] The `:genre`, `:label` `:author` should be set as `attr_reader` because you need to create a custom setter method for them.

Reply:
done, thanks!